### PR TITLE
add default config, change config load, fix output mode

### PR DESCRIPTION
**Pull request by yhung119**
Mostly updating way to load config to make it more user friendly
- If user input non-exist config, we will use default config.
- Instead of user give configname in the current directory, we can load in other folders.
- Output mode should not include APPEND

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -33,10 +33,22 @@ func LoadConfigs(configName string) *PeacemakrConfig {
...
+		configuration = PeacemakrConfig{
+				Verbose: true,
+				Host: "https://api.peacemakr.io",
+				PersisterFileLocation: "/tmp/",
+				ClientName: "peacemakr-cli",
+				ApiKey: viper.GetString("ApiKey"),
+		}
+
+		if configuration.Verbose {
+			log.Printf("Config:\n Verbose: %v\n Host: %v\n Persister file location: %v\n Client Name: %v\n",  configuration.Verbose, configuration.Host, configuration.PersisterFileLocation,  configuration.ClientName)
```
Comment (notasecret) [inline main.go]: perfect

File: main.go [hunk 2]
```diff
@@ -167,33 +179,41 @@ func (l *CustomLogger) Printf(format string, args ...interface{}) {
...
+	if _, err := os.Stat(config.PersisterFileLocation); os.IsNotExist(err) {
+		os.MkdirAll(config.PersisterFileLocation, os.ModePerm)
+	}
+
 	sdk, err := peacemakr_go_sdk.GetPeacemakrSDK(
 		config.ApiKey,
 		config.ClientName,
 		&config.Host,
-		utils.GetDiskPersister("/tmp/"),
+		utils.GetDiskPersister(config.PersisterFileLocation),
```
Comment (notasecret) [inline main.go]: nice

File: main.go [hunk 3]
```diff
@@ -167,33 +179,41 @@ func (l *CustomLogger) Printf(format string, args ...interface{}) {
...
+		log.Println("Finish parsing flag and config")
+		log.Printf("inputfilename: %s, OutputFilename: %s", *inputFileName, *outputFileName)
+	}
 
 	if config.Verbose {
 		log.Println("Setting up SDK...")
 	}
 
+	if _, err := os.Stat(config.PersisterFileLocation); os.IsNotExist(err) {
+		os.MkdirAll(config.PersisterFileLocation, os.ModePerm)
```
Comment (notasecret) [inline main.go]: yeeee, many thx :)

**Comment by yhung119** [comment]
thanks for the review :)

**Pull request merged**